tests/tests/lightspeed-install.cy.ts

```diff
@@ -517,7 +517,7 @@ spec:
       cy.get(modal).find('button').contains('Cancel').click();
       cy.get(modal).find('button').contains('Edit').click();
       cy.get(modal)
-        .find('.ols-plugin__code-block__title')
+        .find('.pf-v6-c-code-editor__code')
```

Comment:
I'm trying to avoid using `pf-v6-*` classes in the tests because
- They will need to be updated the next time we upgrade PatternFly
- We won't be able to cherry pick this change back to the PatternFly 5 branch

Reply:
Actually, I think the `.ols-plugin__code-block__title` really should pass (should contain the expected pod name), so if this is failing it might indicate an actual bug.

Reply:
tbh @kyoto , I'm fairly sure this is happening because unless we search for a value within the monaco-editor, there is a second of load time for the monaco-editor where we try to input into the textarea. Then when we check that it got correctly edited, the value isn't there.
It only happens the first time around you try to attach something. I will prepare a workaround for it in this PR but could you check this please?

Reply:
👍